app/services/client.py

```diff
@@ -181,27 +171,68 @@ def intake(client_data: ClientInTake, doctor: Doctor) -> ClientInfo:
             client_in_queue,
         )
 
-        if not client_in_queue:
-            raise HTTPException(
-                status_code=status.HTTP_405_METHOD_NOT_ALLOWED,
-                detail="Client doesn't at queue",
+        visits = Visit.query.filter(
+            and_(
+                Visit.date == today,
+                Visit.client_id == client.id,
+                Visit.end_time == None,  # noqa E711
             )
+        ).all()
 
-        if client_in_queue:
-            client_in_queue.visit_id = visit.id
-            client_in_queue.canceled = True
-            client_in_queue.save(True)
+        all_visits = Visit.query.filter(
+            and_(
+                Visit.date == today,
+                Visit.client_id == client.id,
+            )
+        ).all()
+
+        if not visits:
+            visit = Visit(
+                date=today,
+                start_time=datetime.datetime.now(),
+                # TODO: rougue_mode
```

Comment:
Do we still need this comment?

Reply:
So I still need this comment, this is my next task)